cppForSwig/BlockDataManagerConfig.cpp

```diff
@@ -20,6 +20,20 @@
 
 using namespace std;
 
+size_t MAX_THREADS()
+{
+	size_t cpu_threads = std::thread::hardware_concurrency();
+
+	if (!cpu_threads)
+		return 1;
```

Comment:
Please use indent interval as in the rest of the code (3 spaces).

Reply:
Fixed sorry, I installed a vim plugin vim-sleuth to detect indents.